build.gradle.kts

```diff
@@ -33,7 +33,7 @@ dependencies {
 
     //K9-format
     implementation("no.nav.k9:soknad:$k9FormatVersion")
-    implementation("org.glassfish:jakarta.el:3.0.3")
+    implementation("org.glassfish:jakarta.el:3.0.4")
```

Comment:
Kan denne slettes?

Reply:
Ah kan den ? :D Trodde den var nødvendig for k9-format validering

Reply:
Ser lovende ut

Reply:
Fungerte

Reply:
Nice 👍